server/neptune/workflows/internal/activities/job.go

```diff
@@ -20,7 +20,7 @@ type CloseJobRequest struct {
 }
 
 func (t *jobActivities) CloseJob(ctx context.Context, request CloseJobRequest) error {
-	err := t.StreamCloser.Close(ctx, request.JobID)
+	err := t.StreamCloser.CloseJob(ctx, request.JobID)
 	if err != nil {
 		logger.Error(ctx, errors.Wrapf(err, "closing job").Error())
 	}
```

Comment:
this log doesn't really seem useful if you're returning the error.

Reply:
The reason I chose to log here was because this activity is called inside a deferred function after a `Plan` or `Apply` job is completed. Since we can't propagate the errors back up,  I decided to log the error inside the activity.